src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -43,16 +43,16 @@
     <Compile Include="System\Text\SBCSCodePageEncoding.cs" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
-    <Compile Include="System\Text\CodePagesEncodingProvider.Windows.cs" />
-    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.Encoding.Constants.cs">
-      <Link>Common\Interop\Windows\kernel32\Interop.Encoding.Constants.cs</Link>
-    </Compile>
+    <Compile Include="System\Text\CodePagesEncodingProvider.Windows.cs" />    
     <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.GetCPInfoEx.cs">
       <Link>Common\Interop\Windows\Interop.GetCPInfoEx.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Kernel32\Interop.WideCharToMultiByte.cs">
```

Comment:
@jkotas constants are used only here...i removed file on corefx and added reference to corelib. Is this a good  compromise to you?(If not i can create a Interop.Encoding.Constants.cs on corecrl, remove from existent cs and after merge on corefx change ref here)